template_feed/Microsoft.DotNet.Common.ItemTemplates/content/ViewImports/.template.config/template.json

```diff
@@ -0,0 +1,27 @@
+{
+  "author": "Microsoft",
+  "classifications": [
+    "Web",
+    "ASP.NET"
+  ],
+  "name": "MVC ViewImports",
+  "generatorVersions": "[1.0.0.0-*)",
+  "groupIdentity": "Microsoft.AspNetCore.Mvc.ViewImports",
+  "precedence": "100",
+  "identity": "Microsoft.AspNetCore.Mvc.ViewImports",
+  "shortName": "viewimports",
+  "sourceName": "_ViewImports",
+  "primaryOutputs": [
+    {
+      "path": "_ViewImports.cshtml"
+    }
+  ],
+  "defaultName": "_ViewImports",
```

Comment:
I'm a bit confused. The only file in this template (other change the config json files) is called _ViewImports.cshtml. Do you mean that it's getting its name changed to content.cshtml?

Can you share the exact command line that you're using to invoke the template?